libs/ledger-live-common/src/api/Ethereum.ts

```diff
@@ -310,9 +329,9 @@ export const apiForCurrency = (currency: CryptoCurrency): API => {
       delete post.gasPrice;
       const { data } = await network({
         method: "POST",
-        url: `${baseURL}/addresses/${address}/estimate-gas-limit`,
+        url: `https://explorers.api-01.live.ledger-stg.com/blockchain/v3/eth/addresses/${address}/estimate-gas-limit`,
```

Comment:
it seems still hardcoded here which will be a problem for non ETH coins (api/Ethereum is used not only by 'ethereum' coin)

if this is a temporary thing, please make the PR a "Draft" as this would be a remaining WIP